### PR TITLE
fix(material-experimental/mdc-slider): init step on thumb inputs

### PR DESCRIPTION
**Pull request by wagnermaciel**

### CODE REVIEW

**Comment by wagnermaciel** [comment]
@mmalerba Small fix here. It seems in refactoring the slider thumb input initialization logic I overlooked the `step` property altogether

File: src/material-experimental/mdc-slider/slider.ts [hunk 1]
```diff
@@ -164,6 +164,7 @@ export class MatSliderThumb implements AfterViewInit {
       : this._slider.max;
     this._elementRef.nativeElement.min = `${min}`;
     this._elementRef.nativeElement.max = `${max}`;
+    this._elementRef.nativeElement.step = `${this._slider.step}`;
```
Comment (crisbeto) [inline src/material-experimental/mdc-slider/slider.ts:167]: nit: `this._elementRef.nativeElement` is being repeated in several places. Would it make sense to move it out into a variable?

File: src/material-experimental/mdc-slider/slider.ts [hunk 2]
```diff
@@ -155,7 +155,7 @@ export class MatSliderThumb implements AfterViewInit {
    * instead be capped at either the default min or max.
    *
    */
-  private _initializeInputMinMax(): void {
+  private _initializeInputMinMaxAndStep(): void {
```
Comment (crisbeto) [inline src/material-experimental/mdc-slider/slider.ts:158]: Maybe just call this `_initializeInputState` so the name doesn't have to be updated any time we add/remove a property?

**Review by crisbeto — APPROVED** [review]
LGTM

**Pull request merged**